app/src/main/java/nu/yona/app/api/manager/dao/ActivityCategoriesDAO.java

```diff
@@ -10,69 +10,64 @@
 
 package nu.yona.app.api.manager.dao;
 
+import android.content.ContentValues;
 import android.content.Context;
 import android.database.Cursor;
 import android.database.sqlite.SQLiteOpenHelper;
 import android.util.Log;
 
-import java.util.ArrayList;
-import java.util.List;
-
 import nu.yona.app.api.db.DBConstant;
-import nu.yona.app.api.model.ErrorMessage;
-import nu.yona.app.api.model.User;
+import nu.yona.app.api.model.ActivityCategories;
 import nu.yona.app.listener.DataLoadListener;
-import nu.yona.app.listener.DataLoader;
 
 /**
  * Created by kinnarvasa on 14/04/16.
  */
 public class ActivityCategoriesDAO extends BaseDAO {
 
+    private final String ID = "1";
```

Comment:
we are using this for updating that Category object into database , as what we are doing same for User.